Improved output messages to be clearer.

AndyHeap-NeoTech opened the pull request:
This will highlight if there is an empty message.

AndyHeap-NeoTech committed:
Improved output messages to be clearer.

This will highlight if there is an empty message.

fbiville commented on Neo4j.Driver/Neo4j.Driver.Tests.Integration/Reactive/NestedQueriesIT.cs:
```diff
@@ -78,8 +78,8 @@ public void ShouldCloseSessionWhenRunNestedQueriesWithSessionRuns()
-			Output.WriteLine(message);
-			return message.Contains(expectedMessage);
+			Output.WriteLine("Actual Message: " + message);
+			return message.Contains("Excpeted Message: " + expectedMessage);
```
```suggestion
			return message.Contains("Expected Message: " + expectedMessage);

```

AndyHeap-NeoTech committed:
Update Neo4j.Driver/Neo4j.Driver.Tests.Integration/Reactive/NestedQueriesIT.cs

Co-authored-by: Florent Biville <445792+fbiville@users.noreply.github.com>

AndyHeap-NeoTech committed:
Fix for ShouldErrorToRunNestedQueriesWithTransactionFunctions

Error in which string sub-string was looked for in the error message which broke the test.
